{{cookiecutter.project_slug}}/Makefile

```diff
@@ -433,21 +443,17 @@ hypertrain: _check_setup wandb-check-auth   ### Run jobs in parallel for hyperpa
 	done; \
 	echo -e "\nStarted $(N_HYPERPARAM_JOBS) hyper-parameter search jobs of sweep '$$sweep'.\nUse 'neuro ps' and 'neuro status <job>' to check."
 
+.PHONY: kill-hypertrain
+kill-hypertrain:  ### Terminate hyper-parameter search training jobs of the latest sweep
+	sweep=`tail -1 $(WANDB_SWEEPS_FILE)` && \
+	jobs=`neuro -q ps --tag "target:hypertrain" --tag "sweep:$$sweep" $(_PROJECT_TAGS) | tr -d "\r"` && \
```

Comment:
We also agreed to change `sweep` to `wandb-sweep`.

Reply:
oops sorry missed that